ts/components/conversation/message/message-item/GenericReadableMessage.tsx

```diff
@@ -83,6 +85,9 @@ export const GenericReadableMessage = (props: Props) => {
 
   const handleContextMenu = useCallback(
     (e: MouseEvent<HTMLElement>) => {
+      if (legacyGroupIsDeprecated) {
```

Comment:
Going to discuss this with you.

Reply:
discussed? waiting on internal discussion